sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -0,0 +1,141 @@
+    <ListingContext.Provider value={listing}>
+      <ListingGuard>
+        <Layout>
+          <Head>
+            <title>{t("nav.siteTitlePartners")}</title>
+          </Head>
+
+          <MetaTags
+            title={t("nav.siteTitlePartners")}
+            image={metaImage}
+            description={metaDescription}
+          />
+
+          <NavigationHeader
+            title={listing.name}
+            listingId={listing.id}
+            tabs={{
+              show: true,
+              listingLabel: t("t.listingSingle"),
+              applicationsLabel: t("nav.applications"),
+              lotteryLabel:
+                listing.status === ListingsStatusEnum.closed &&
+                listing.reviewOrderType === ReviewOrderTypeEnum.lottery
+                  ? t("listings.lotteryTitle")
+                  : undefined,
+            }}
+            breadcrumbs={
+              <Breadcrumbs>
+                <BreadcrumbLink href="/">{t("t.listing")}</BreadcrumbLink>
+                <BreadcrumbLink href={`/listings/${listing.id}`}>{listing.name}</BreadcrumbLink>
+                <BreadcrumbLink href={`/listings/${listing.id}/lottery`} current>
+                  {t("listings.lotteryTitle")}
+                </BreadcrumbLink>
+              </Breadcrumbs>
+            }
+          />
+
+          <ListingStatusBar status={listing?.status} />
+          <section className={styles["lottery"]}>
+            <div className={styles["parent"]}>
+              <div className={styles["container"]}>
+                <div className={styles["main"]}>
+                  <Card spacing={"lg"}>
+                    <CardSection>
+                      <Icon icon={faTicket} size="2xl" />
+                      <Heading priority={2} size={"2xl"}>
+                        {t("listings.lottery.noData")}
+                      </Heading>
+                      <div className={styles["card-description"]}>
+                        {t("listings.lottery.noDataDescription")}
+                      </div>
+                      <div>
+                        <Button>{t("listings.lottery.runLottery")}</Button>
+                      </div>
+                    </CardSection>
+                  </Card>
+                </div>
+                <aside className={styles["side"]}>
+                  <Card>
+                    <CardHeader>
+                      <Heading priority={2} size={"lg"}>
+                        {t("listings.lottery.history")}
+                      </Heading>
+                    </CardHeader>
+                    <CardSection>
+                      {getHistoryItem(
+                        "November 21st, 2023 at 8:30am",
+                        "Listing closed",
+                        "By property"
+                      )}
+                      {getHistoryItem(
+                        "November 21st, 2023 at 8:30am",
+                        "Listing closed",
+                        "By property"
+                      )}
```

Comment:
Duplicate items. This will pull from listing data in the future yeah?

Reply:
Yep yep, just wanted more than one to make sure the padding looked ok

Reply:
This dummy data scared me for a moment